uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/apis/leave-request-api.js

```diff
@@ -183,6 +183,33 @@ define([
         },
 
         /**
+         * Get leave request for the given id
+         *
+         * @param {object} id - leave request id
+         *
```

Comment:
not sure why you are adding this empty space between @param and @return, it shouldn't be here